### ISSUE TITLE
Duplicate column headers

### ISSUE THREAD

**Issue by ohthehugemanatee** [cjhowald/ynab-csv#14]
My import source has a duplicate column header: "amount" appears twice. If I select either "amount" column in the converter, it only displays the values from the second "amount" column. 

I know, this is a violation of whatever spec we could imagine for CSV. Still, it happens. Perhaps in parsing we could append a _1 to the end of duplicate column titles or something.

**Comment by cjhowald** [comment]
Good catch. I have the same situation, but the second "amount" is the EUR value, so it is what I want. I will look at how papaparse parses it with duplicate headers and see if they can be differentiated.
